js/search.js

```diff
@@ -949,25 +949,38 @@
 			for (var i = 0; i < moves.length; i++) {
 				var id = moves[i];
 				var isViable = BattleMovedex[id] && BattleMovedex[id].isViable;
-				if (id === 'aerialace') isViable = (toId(set.ability) === 'technician');
+				if (id === 'aerialace') isViable = (toId(set.species) in {scyther:1, aerodactylmega:1, kricketune:1});
 				if (id === 'bellydrum') isViable = (toId(set.species) in {azumarill:1, linoone:1, slurpuff:1});
-				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1});
+				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1, alakazam:1});
+				if (id === 'curse') isViable = (toId(set.species) === 'snorlax');
+				if (id === 'drainingkiss') isViable = (toId(set.ability) === 'triage');
 				if (id === 'dynamicpunch') isViable = (toId(set.ability) === 'noguard');
+				if (id === 'electroball') isViable = (toId(set.species) === 'raichualola');
+				if (id === 'endeavor') isViable = (toId(set.species) in {mamoswine:1, scolipede:1});
```

Comment:
...would not manually override this; anything can run a FEAR set...

Reply:
I see why taking out mamo but Scolipede doesn't need to fear w/ access to speed boost and spikes so i find it having more merit

Reply:
I mean, I think Endeavor should just always be usable.

Reply:
that's fine with me, I'm planning a req for the always/never usabilities and I'll put it in there